pr_agent/settings/configuration.toml

```diff
@@ -1,8 +1,8 @@
 [config]
 # models
-model_weak="gpt-4o-mini-2024-07-18"
 model="gpt-4o-2024-11-20"
 fallback_models=["gpt-4o-2024-08-06"]
+#model_weak="gpt-4o-mini-2024-07-18" # optional, a weaker model to use for some easier tasks
```

Comment:
@KennyDizi 
some additional adjustments 1: 
I want the model_weak to be _optional_. only if the user chooses to define it, it will be used. in general, most users prefer quality over cost

Reply:
That's a good point @mrT23. Thanks for the heads up!